docs/resources/fabric_external_connection_policy.md

```diff
@@ -0,0 +1,176 @@
+
+```hcl
+
+resource "aci_fabric_external_connection_policy" "full_example_tenant" {
+  parent_dn    = aci_tenant.example.id
+  annotation   = "annotation"
+  description  = "description_1"
+  id_attribute = "1"
+  name         = "name_1"
+  name_alias   = "name_alias_1"
+  owner_key    = "owner_key_1"
+  owner_tag    = "owner_tag_1"
+  community    = "extended:as2-nn4:5:16"
+  site_id      = "0"
+  peering_profile = {
+    annotation  = "annotation_1"
+    description = "description_1"
+    name        = "name_1"
+    name_alias  = "name_alias_1"
+    owner_key   = "owner_key_1"
+    owner_tag   = "owner_tag_1"
+    password    = "password_1"
+    type        = "automatic_with_full_mesh"
+    annotations = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+    tags = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+  }
+  annotations = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+  tags = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+}
+
+```
+
+All examples for the Fabric External Connection Policy resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_fabric_external_connection_policy) folder.
+
+## Schema ##
+
+### Required ###
+
+* `id_attribute` (id) - (string) The identifier of the Fabric External Connection Policy object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the Fabric External Connection Policy object.
+
+### Optional ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+  - Default: `uni/tn-infra`
+* `annotation` (annotation) - (string) The annotation of the Fabric External Connection Policy object. This attribute is supported in ACI versions: 3.2(1l) and later.
+  - Default: `orchestrator:terraform`
+* `description` (descr) - (string) The description of the Fabric External Connection Policy object.
+* `name` (name) - (string) The name of the Fabric External Connection Policy object.
+* `name_alias` (nameAlias) - (string) The name alias of the Fabric External Connection Policy object. This attribute is supported in ACI versions: 2.2(1k) and later.
+* `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+* `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+* `community` (rt) - (string) A global route target that is used to define the communities for route leaking or redistribution. It can be used in a multi-site deployment to manage routing policies across fabrics.
```

Comment:
I think we should adjust the sentence here because the community is also used for multi-pod scenario. Also not sure if we should use `It can be`, likely we should rephrase this into something more clear. 

See also: https://www.cisco.com/c/en/us/solutions/collateral/data-center-virtualization/application-centric-infrastructure/white-paper-c11-739714.html